### DISCUSSION TITLE
Are old import configurations still supported?

### DISCUSSION THREAD

**Discussion by zjean** [firefly-iii/firefly-iii#5734] [Firefly III]
Hi,
Using the old csv importer, I used the following import configuration for my Rabobank account.
`{
    "file-type": "csv",
    "date-format": "Y-m-d",
    "has-headers": true,
    "delimiter": ",",
    "apply-rules": true,
    "specifics": [],
    "import-account": 1,
    "column-count": 26,
    "column-roles": [
        "account-iban",
        "currency-code",
        "account-bic",
        "internal-reference",
        "date-transaction",
        "date-interest",
        "amount",
        "_ignore",
        "opposing-iban",
        "opposing-name",
        "_ignore",
        "note",
        "opposing-bic",
        "_ignore",
        "sepa-batch-id",
        "sepa-ct-id",
        "sepa-db",
        "sepa-ct-op",
        "external-id",
        "description",
        "description",
        "description",
        "note",
        "amount_foreign",
        "foreign-currency-code",
        "note"
    ],
    "column-do-mapping": [
        true,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        true,
        true,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false,
        false
    ],
    "column-mapping-config": {}
}`

When using the new data importer, and selecting the existing import configuration (from the drop down), the Role configuration screen is empty. All roles are set to ignore this column instead of using the settings from the import configuration file.
The same happens when I use the [example file](https://github.com/firefly-iii/import-configurations/blob/main/nl/rabobank/rabobank-new-csv-format.json)

**Comment by JC5** [comment]
They should be yes, but I haven't tested them extensively in ages. I'll pick it up.
